django/contrib/admin/templates/admin/filter.html

```diff
@@ -1,8 +1,12 @@
 {% load i18n %}
```

Comment:
I think it would be better to have a configurable number of items to decide if the filter is open or closed.
In this first PR perhaps I would remove the condition on the number of elements to manage in a separate PR.

Reply:
Excellent! Do you have any code example about another configurable variable (to have as reference)?